fw/http_msg.c

```diff
@@ -815,6 +820,33 @@ __hdr_sub(TfwHttpMsg *hm, const TfwStr *hdr, unsigned int hid)
 	return TFW_PASS;
 }
 
+/**
+ * Substitute request method with "get"
```

Comment:
Personally, I am used to treat the term `sub` in the code as one in the series of `add/sub/mul/del`. If the goal is a shortcut name for `substitution`, perhaps a better and more common term would be `subst`. Just a thought.

Reply:
ok. done.